### ISSUE TITLE
Citrix Client

### ISSUE THREAD

**Issue by scrocker** [pywinauto/pywinauto#535]
I'm able to connect to the Citrix Receiver is there a way to connect to the application running in Citrix?

**Comment by vasily-v-ryabov** [comment]
No, because all virtual desktops don't provide application specific info about UI elements. You have to copy the script to remote machine.

**Comment by mayurpande** [comment]
When you mean the script, you mean the python script? So for example I have a vmware with my citrix environment, after I login, I have a program that I want to run and automate, is this possible?

**Comment by vasily-v-ryabov** [comment]
It's possible but not so trivial. First I'd recommend learning the [Remote Execution Guide](https://pywinauto.readthedocs.io/en/latest/remote_execution.html) in details. Any virtual machine can be considered as a remote PC. So all things are applicable the same way. Probably you need to set up NAT virtual network adapter for VMWare properly to get network access to your virtual machine OS.
